docs/reference/contrib/settings.rst

```diff
@@ -264,5 +266,55 @@ and two more to fetch each page):
 
 .. code-block:: html
 
-    {{ settings.app_label.ImportantPages.donate_page.url }}
-    {{ settings.app_label.ImportantPages.sign_up_page.url }}
+    {% load wagtailcore_tags %}
+    {% page_url settings.app_label.ImportantPages.donate_page %}
+    {% page_url settings.app_label.ImportantPages.sign_up_page %}
```

Comment:
Note: this change will actually not work, the template tag is `pageurl` not `page_url`
see: wagtail/core/templatetags/wagtailcore_tags.py